lib/client.js

```diff
@@ -60,6 +60,16 @@ var clientOptions = require('./client-options');
  *  Setting it to false will cause less overhead and the reference of the network buffer to be maintained until the row / result set are de-referenced.
  *  Default: true.
  * </p>
+ * @property {Boolean} encoding.useUndefinedAsUnset Valid for Cassandra 2.2 and above. Determines that, if a parameter is set to
```

Comment:
It would be nice to have but to do that we would have to validate at client level after connect, another validation method. Also we would still have to validate at encoding level if the user is using `unset` with a lower protocol version.